docs/DeveloperGuide.md

```diff
@@ -390,19 +393,46 @@ Step 1. The user launches the application today.
 
 Step 2. The user executes `suggest` command, which creates a `SuggestCommand` object.
 
-Step 3. The `SuggestCommand` created is executed by `SuggestCommand#execute()`
+Step 3. The `SuggestCommand` created is executed by `SuggestCommand#execute()`.
 
 Step 4. `SuggestCommand#execute()` then calls the following methods from `Model`:
 
 * `Model#isUserCreated()`
 * `Model#calculateSuggestedCalorie()`
 
-Step 5. `SuggestCommand#execute()` returns a `CommandResult` that displays an estimated suggested amount of calories the user should consume per day.
+Step 5. `SuggestCommand#execute()` returns a `CommandResult` that displays an estimated suggested amount of calories 
+the user should consume per day.
```

Comment:
```suggestion
Step 5. `SuggestCommand#execute()` returns a `CommandResult` that displays an estimated suggested amount of calories the user should consume per day.
```

I think it would be better to put it all in a one line 🤔